build-all.sh

```diff
@@ -18,21 +18,43 @@
 # THIS SCRIPT IS INTENDED FOR LOCAL DEV USAGE ONLY
 #
 
+SCALA_VERSIONS=(2.11 2.12 2.13)
+
 BASE_DIR=$(dirname "$0")
+MODULE_DIRS=$(find "$BASE_DIR" -type f -name "pom.xml" -printf '%h\n')
+MVN_EXEC="mvn"
+
+print_title() {
+  echo "░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░"
+  echo "                           $1                                                  "
+  echo "░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░"
+}
 
 cross_build() {
-  SCALA_VER=$1
-  echo "Building with Scala $SCALA_VER"
-  find $BASE_DIR/target/* -type d -exec rm -rf {} \;
-  mvn scala-cross-build:change-version -Pscala-$SCALA_VER
-  mvn install
+  bin_ver=$1
+
+  # pre-cleaning
+  for dir in $MODULE_DIRS; do
+    rm -rf "$dir"/target
+  done
```

Comment:
Why not `mvn clean`?

Reply:
It's just easier and more robust :) `mvn clean` only affects modules included in the reactor that in turn is configured by profiles, so just running `mvn clean` wouldn't actually clean everything, but here what I want is a total wipeout.

Reply:
Ah, and yes, if you wonder what does it have to do with modules in a single-module project, I forgot to mention that the script was just copy-pasted from another project which is multi-module one :) If commons is broken into modules one day the script should just continue working as expected.